src/main/java/ee/buerokratt/ruuter/domain/steps/http/HttpStep.java

```diff
@@ -76,5 +72,17 @@ protected void logStep(Long elapsedTime, ConfigurationInstance ci) {
         LoggingUtils.logStep(log, this, ci.getRequestOrigin(), elapsedTime, args.getUrl(), requestContent, responseContent, String.valueOf(responseStatus));
     }
 
+    private ResponseEntity<Object> tryGetRequestResponse(ConfigurationInstance ci) {
+        try {
```

Comment:
Although it's more "compact" here, i think the try-catch should be in the http helper, considering we use the helper outside of steps aswell, and want it to behave the same way